src/extensions/default/Inline3DParametersEditor/main.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2012 - present Adobe Systems Incorporated. All rights reserved.
```

Comment:
Based on [this](https://github.com/mozilla/brackets/blob/master/src/extensions/default/InlineColorEditor/main.js) file